.github/workflows/update-binaries.yaml

```diff
@@ -0,0 +1,47 @@
+name: Update bin.walrus.site with latest binaries
+
+on:
+  # every week
+  schedule:
+    - cron: '14 3 * * 0'
```

Comment:
Question: this is good for now, but could we eventually tie this to the actual generation of the new binaries?

Reply:
I agree that this would be great, I will have to look into this. 

@quepcompute Is there a straight-forward way of calling this workflow from the https://github.com/MystenLabs/sui-operations/blob/main/.github/workflows/continuous-walrus.yml workflow?

I will still merge this as it is now, we can then add an additional trigger to the workflow later.

Reply:
Off the top of my head, I think that this is doable.  I'll look into this early next week.

Reply:
I think this can be done with `repository_dispatch` [events](https://www.nieknijland.nl/blog/how-to-trigger-a-github-action-workflow-in-another-repository).  Making sure I have the full context here, when would we want this action to be called from the other workflow?  For every run on the  devnet branch since thats the one that builds and publishes the binaries?

Reply:
> I think this can be done with repository_dispatch [events](https://www.nieknijland.nl/blog/how-to-trigger-a-github-action-workflow-in-another-repository).

I also looked into the `repository_dispatch` trigger, and it seems to do what we want. It's a bit unfortunate that we need a personal access token in the calling workflow. How do we usually handle this?

> when would we want this action to be called from the other workflow? For every run on the devnet branch since thats the one that builds and publishes the binaries?

Yes, exactly.